test/custom_task_test.go

```diff
@@ -122,11 +126,25 @@ spec:
 	}
 
 	// Get the Run name.
-	if len(pr.Status.Runs) != 2 {
-		t.Fatalf("PipelineRun had unexpected .status.runs; got %d, want 2", len(pr.Status.Runs))
+	var runNames []string
+	if embeddedStatusValue != config.MinimalEmbeddedStatus {
```

Comment:
I think the branches are off here, you probably want
```
if embeddedStatusValue != config.MinimalEmbeddedStatus {}
if embeddedStatusValue != config.FullEmbeddedStatus{}
```
both here and below (ensuring config.BothEmbeddedStatus is tested correctly)

Reply:
Good catch!

Reply:
same thing on line 218 and in custom_task_test

Reply:
Well I feel a silly man. Fixed.